timesketch/frontend-ng/src/components/Visualization/VisualizationEditor.vue

```diff
@@ -0,0 +1,418 @@
+                @updateField="selectedField = $event"
+                :aggregator="selectedAggregator"
+                @updateAggregator="selectedAggregator = $event"
+                :metric="selectedMetric"
+                @updateMetric="selectedMetric = $event"
+                :maxItems="selectedMaxItems"
+                @updateMaxItems="selectedMaxItems = $event"
+                :interval="selectedInterval"
+                @updateInterval="selectedInterval = $event"
+                :intervalQuantity="selectedIntervalQuantity"
+                @updateIntervalQuantity="selectedIntervalQuantity = $event"
+                :splitByTimeline="selectedSplitByTimeline"
+                @updateSplitByTimeline="selectedSplitByTimeline = $event"
+              ></TsAggregationConfig>
+              <TsChartConfig
+                :aggregatorType="selectedAggregator"
+                :chartType="selectedChartType"
+                @updateChartType="selectedChartType = $event"
+                :title="selectedChartTitle"
+                @updateTitle="selectedChartTitle = $event"
+                :height="selectedHeight"
+                @updateHeight="selectedHeight = $event"
+                :width="selectedWidth"
+                @updateWidth="selectedWidth = $event"
+                :xTitle="selectedXTitle"
+                @updateXTitle="selectedXTitle = $event"
+                :showXLabels="selectedShowXLabels"
+                @updateShowXLabels="selectedShowXLabels = $event"
+                :yTitle="selectedYTitle"
+                @updateYTitle="selectedYTitle = $event"
+                :showYLabels="selectedShowYLabels"
+                @updateShowYLabels="selectedShowYLabels = $event"
+                :showDataLabels="selectedShowDataLabels"
+                @updateShowDataLabels="selectedShowDataLabels = $event"
+              ></TsChartConfig>
+            </v-card-text>
+          </v-card>
+        </v-col>
+        <v-col cols="8">
+          <v-card outlined>
+            <v-card-text>
+              <TsChartCard
+                v-if="chartSeries && selectedChartType"
+                :fieldName="selectedField.field"
+                :metricName="selectedMetric"
+                :is-time-series="selectedAggregator ? selectedAggregator.endsWith('date_histogram') : false"
+                :chartSeries="chartSeries" 
+                :chartLabels="chartLabels"
+                :chartTitle="selectedChartTitle"
+                :chartType="selectedChartType"
+                :height="selectedHeight"
+                :width="selectedWidth"
+                :xTitle="selectedXTitle"
+                :showXLabels="selectedShowXLabels"
+                :yTitle="selectedYTitle"
+                :showYLabels="selectedShowYLabels"
+                :showDataLabels="selectedShowDataLabels"
+              ></TsChartCard>
+            </v-card-text>
+          </v-card>
+        </v-col>
+      </v-row>
+    </v-card-text>
+    <v-card-actions>
```

Comment:
What do you think about moving the actions into the config card? In the end the user wants to save this specific viz config they just created.

Reply:
I'll message you offline on this.